File: examples/docs-code-snippets/src/main/java/examples/java/application/ManualResolveInteraction.java
package examples.java.application;

import com.ing.baker.runtime.javadsl.Baker;
import com.ing.baker.runtime.javadsl.EventInstance;

public class ManualResolveInteraction {

    public void resolveExample(Baker baker, String recipeInstanceId) {
        baker.resolveInteraction(
            recipeInstanceId,
            "ShipOrder",
            EventInstance.from("ShippingFailed")
        );
    }
}

File: examples/docs-code-snippets/src/main/kotlin/examples/kotlin/application/ManualResolveInteraction.kt
package examples.kotlin.application

import com.ing.baker.runtime.javadsl.EventInstance
import com.ing.baker.runtime.kotlindsl.Baker

suspend fun resolveExample(baker: Baker, recipeInstanceId: String) {
    baker.resolveInteraction(
        recipeInstanceId,
        "ShipOrder",
        EventInstance.from("ShippingFailed")
    )
}